cmd/atlas/internal/cmdapi/schema.go

```diff
@@ -84,33 +73,177 @@ the project file (see: https://atlasgo.io/cli/projects).
 
 If run with the "--dry-run" flag, atlas will exit after printing out the planned
 migration.`,
-		PreRunE: schemaFlagsFromEnv,
-		RunE:    CmdApplyRun,
-		Example: `  atlas schema apply -u "mysql://user:pass@localhost/dbname" -f atlas.hcl
+			PreRunE: schemaFlagsFromEnv,
+			Example: `  atlas schema apply -u "mysql://user:pass@localhost/dbname" -f atlas.hcl
   atlas schema apply -u "mysql://localhost" -f schema.hcl --schema prod --schema staging
   atlas schema apply -u "mysql://user:pass@localhost:3306/dbname" -f schema.hcl --dry-run
   atlas schema apply -u "mariadb://user:pass@localhost:3306/dbname" -f schema.hcl
   atlas schema apply --url "postgres://user:pass@host:port/dbname?sslmode=disable" -f schema.hcl
   atlas schema apply -u "sqlite://file:ex1.db?_fk=1" -f schema.hcl`,
+		}
+	)
+	cmd.Flags().SortFlags = false
+	cmd.Flags().StringSliceVarP(&flags.Paths, flagFile, "f", nil, "[paths...] file or directory containing the HCL files")
+	addFlagURL(cmd.Flags(), &flags.URL)
+	addFlagExclude(cmd.Flags(), &flags.Exclude)
+	addFlagSchema(cmd.Flags(), &flags.Schemas)
+	addFlagDevURL(cmd.Flags(), &flags.DevURL)
+	addFlagDryRun(cmd.Flags(), &flags.DryRun)
+	addFlagAutoApprove(cmd.Flags(), &flags.AutoApprove)
+	addFlagDSN(cmd.Flags(), &flags.DSN)
+	cobra.CheckErr(cmd.MarkFlagRequired(flagURL))
+	cobra.CheckErr(cmd.MarkFlagRequired(flagFile))
+
+	cmd.RunE = func(cmd *cobra.Command, _ []string) error {
```

Comment:
I find it less readable and friendly to maintain command functions this way. Can we move it out to the top level, please?